manifests/postgresql.crd.yaml

```diff
@@ -0,0 +1,312 @@
+                  type: string
+                  pattern: '\^([0-9]+)-(0[1-9]|1[012])-(0[1-9]|[12][0-9]|3[01])[Tt]([01][0-9]|2[0-3]):([0-5][0-9]):([0-5][0-9]|60)(\\.[0-9]+)?(([Zz])|([+-]([01][0-9]|2[0-3]):[0-5][0-9]))$'
+                  # The regexp matches the date-time format (RFC 3339 Section 5.6) that specifies a timezone as an offset relative to UTC
+                  # Example: 1996-12-19T16:39:57-08:00
+                  # Note: this field requires a timezone
+                uid:
+                  format: uuid
+                  type: string
+            databases:
+              type: object
+              additionalProperties:
+                type: string
+              # Note: usernames specified here as database owners must be declared in the users key of the spec key.
+            dockerImage:
+              type: string
+            enableLogicalBackup:
+              type: boolean
+            enableMasterLoadBalancer:
+              type: boolean
+            enableReplicaLoadBalancer:
+              type: boolean
+            enableShmVolume:
+              type: boolean
+            logicalBackupSchedule:
+              type: string
+              pattern: '^(\d+|\*)(/\d+)?(\s+(\d+|\*)(/\d+)?){4}$'
+            maintenanceWindows:
+              type: array
+              items:
+                type: string
+                pattern: '\
+                  ^\
+                  \ *\
+                  (\
+                    (Mon|Tue|Wed|Thu|Fri|Sat|Sun)\
+                    :(2[0-3]|[01]?\\d):([0-5]?\\d)\
+                    -\
+                    (Mon|Tue|Wed|Thu|Fri|Sat|Sun)\
+                    :(2[0-3]|[01]?\\d):([0-5]?\\d)\
+                  )|(\
+                    (2[0-3]|[01]?\\d):([0-5]?\\d)\
+                    -\
+                    (2[0-3]|[01]?\\d):([0-5]?\\d)\
+                  )\
+                  \ *\
+                  $\'
+            numberOfInstances:
+              type: integer
+              minimum: 0
+            patroni:
+              type: object
+              properties:
+                initdb:
+                  type: object
+                  additionalProperties:
+                    type: string
+                pg_hba:
+                  type: array
+                  items:
+                    type: string
+                slots:
+                  type: object
+                  additionalProperties:
+                    type: string
+                ttl:
+                  type: integer
+                loop_wait:
+                  type: integer
+                retry_timeout:
+                  type: integer
+                maximum_lag_on_failover:
+                  type: integer
+            pod_priority_class_name:
+              type: string
+            postgresql:
+              type: object
+              required:
+                - version
+              properties:
+                version:
+                  enum:
+                    - 9.6
```

Comment:
spilo master still has 9.3 9.4 9.5 in OLDVERSIONS in the dockerfile, shouldn't those be here as well?

ref https://github.com/zalando/spilo/blob/master/postgres-appliance/Dockerfile#L55

Reply:
changed and using strings now